app/src/main/java/com/pokemonshowdown/data/ItemDex.java

```diff
@@ -3,83 +3,47 @@
 import android.content.Context;
 import android.util.Log;
 
+import com.paypal.android.sdk.D;
 import com.pokemonshowdown.app.R;
 import com.pokemonshowdown.application.MyApplication;
 
 import org.json.JSONException;
 import org.json.JSONObject;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.util.HashMap;
-import java.util.Iterator;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 public class ItemDex {
     private final static String ITAG = ItemDex.class.getName();
-    private static ItemDex sItemDex;
-    private HashMap<String, String> mItemDexEntries;
+    public final static JSONObject DUMMY_OBJECT;
 
-    private ItemDex(Context appContext) {
-        mItemDexEntries = readFile(appContext);
-    }
-
-    private HashMap<String, String> readFile(Context appContext) {
-        HashMap<String, String> ItemDexEntries = new HashMap<>();
-        String jsonString;
+    static {
         try {
-            InputStream inputStream = appContext.getResources().openRawResource(R.raw.item);
-            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
-            StringBuilder stringBuilder = new StringBuilder();
-
-            String line;
-            while ((line = bufferedReader.readLine()) != null) {
-                stringBuilder.append(line);
-                stringBuilder.append("\n");
-            }
-            jsonString = stringBuilder.toString();
-            inputStream.close();
-
-            JSONObject jsonObject = new JSONObject(jsonString);
-
-            Iterator<String> keys = jsonObject.keys();
-
-            while (keys.hasNext()) {
-                String key = keys.next();
-                JSONObject entry = jsonObject.getJSONObject(key);
-                ItemDexEntries.put(key, entry.toString());
-            }
+            DUMMY_OBJECT = new JSONObject("{\"gen\":1,\"spritenum\":0,\"num\":0,\"name\":\"\",\"id\":\"dummy\",\"desc\":\"\"}");
```

Comment:
I would add something more descriptive. Like name=dummy, description=Item not found, please report for update etc.